implementation.md

```diff
@@ -1534,3 +1562,66 @@ Here is an example ad request:
+ 
+While IAB Tech Lab advises that all partners participating in the programmatic buying and selling of podcast inventory should pass all prescribed information below, this is first and foremost a technical guideline built to outline process and does not supersede individual business relationships established by buyers and sellers. It does, however, fully define how data should be sent.
+
+In the tables below, an added column for “Value” is used to define the expected information in the listed field for podcast inventory. This is additional information beyond the description of the field provided formally in OpenRTB and helps to reduce ambiguity about how to use the field to signal podcast inventory.
+
+### 7.11.1 Identity Signals in the Device Object
+
+Programmatic advertising is an operational tool built to simplify the process of buying and selling inventory. To accomplish that simplicity, it is important not only to provide as many data signals as possible for a medium, but also to pass them predictably in the prescribed parameters.
+ 
+Identity signals enable all sides of programmatic advertising to augment data, providing a unique value proposition to their clients. Some channels offer additional ID types, such as cookie IDs, device IDs, and other proprietary solutions as a match point. To supplement for these ID’s, podcast ad servers or SSPs should pass a consistent synthetic ID composed of a hashed IP address and Device User Agent to provide a probabilistic alternative for buyers looking for this type of anchor.
+ 
+IAB Tech Lab recommends passing the following values as identified in the OpenRTB spec:
+
+#### Object: Device
+
+|**Field**  |**Value**      |**Type**|**Description**         |
+|:------|:----------|:------------|:------------------------------------------------------------------|
+|ua |User agent |string |A raw user agent string from the browser. See ua field under device object for more details.|
+|sau|User agent|object|Structured user agent as an object detailing device make, model, app, version etc.|
+|ip|Ip address|string|IPv4 address closest to device.|
+|ifa|Device ID|string|For podcast specifically, SSPs should fill in using a persistent synthetic ID.|
+
+#### Signaling ifa_type in the Device Object
+
+The identifier for advertisers (ifa) type is signaled in OpenRTB using [device.ext](https://github.com/ChrisBasis/openrtb/blob/master/extensions/community_extensions/ifa_type.md) to indicate the source of the id (i.e. device, publisher, SSP, or session-based). It does not enumerate platform specific ids such as Roku, Apple, or Android. When the ifa_type is provided by the device (device.ext.ifa_type=dpid), the platform is inferred based on other signals within OpenRTB.
```

Comment:
Should link to other IABTL documentation. It's not a standard if it's not also here.